### PR TITLE
Basics implementation of JWT

### LINKED ISSUE #22: Implement JWT
Currently, Spring Security Session is used.
JWT Tokens should be implemented.

### PR DESCRIPTION
**Pull request by a-merezhanyi**
Implements #22 in the following scope:
- user creation
- user login
- user validation for `tasks` endpoints
- updated Postman collection

Unimplemented:
- refresh token
- revoke token

Appropriate tickets have been created:
- #25 
- #26
- #27
- #29
- #30
- #31
- #35

### CODE REVIEW

File: src/main/java/nextmainfocus/jwt/JwtService.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+
+@Service
+// @ConfigurationProperties("jwt")
+public class JwtService {
+	private static final Logger logger = LoggerFactory.getLogger(JwtService.class);
+
+	private long expiration = 18000000; // 5 hours in milliseconds 5 * 60 * 60 * 1000
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/jwt/JwtService.java:29]: a good practice is it to add _ms to the var's name.

File: src/main/java/nextmainfocus/jwt/JwtService.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	private String secret = "3778217A25432A462D4A614E645267556B58703273357638792F423F4428472B";
+
+	public String authorizationHeader = "Authorization";
```
Comment (merezhanyi) [inline src/main/java/nextmainfocus/jwt/JwtService.java:32]: all such vars could be final (const)

File: src/main/java/nextmainfocus/jwt/JwtService.java (hunk 1 above)
Comment (a-merezhanyi) [reply src/main/java/nextmainfocus/jwt/JwtService.java:29]: Why exactly? ;) Adding any meta-information (typings, allocation, goal, etc.) was a part of [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) and is not considered a best practice in 2023. Btw, CAPS for const in Java is a part of that, currently is an inevitable legacy. 🤗
Btw, this [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) should be moved into an external config file (according to #25)

File: src/main/java/nextmainfocus/jwt/JwtService.java (hunk 2 above)
Comment (a-merezhanyi) [reply src/main/java/nextmainfocus/jwt/JwtService.java:32]: Agree. All such vars should be extracted into an external config file (according to #25)
Today it is "Authorization," tomorrow that might be "authoriSation" 😋

**Pull request merged**